### PR TITLE
Expose half_up rounding in cuDF

### PR DESCRIPTION
**Pull request by shwina**

### CODE REVIEW

**Review by galipremsagar — APPROVED** [review]

**Review by rgsl888prabhu — APPROVED** [review]

File: python/cudf/cudf/tests/test_series.py [hunk 1]
```diff
@@ -692,6 +692,13 @@ def test_series_round(arr, decimals):
     np.array_equal(ser.nullmask.to_array(), result.to_array())
 
 
+def test_series_round_half_up():
+    s = cudf.Series([0.0, 1.0, 1.2, 1.7, 0.5, 1.5, 2.5, None])
+    expect = cudf.Series([0.0, 1.0, 1.0, 2.0, 1.0, 2.0, 3.0, None])
+    got = s.round(how="half_up")
```
Comment (isVoid) [inline python/cudf/cudf/tests/test_series.py:698]: nit: test "half_even" too?

**Review by isVoid — COMMENTED** [review]
Tiny nit, rest is good.

File: python/cudf/cudf/tests/test_series.py (hunk 1 above)
Comment (rgsl888prabhu) [reply python/cudf/cudf/tests/test_series.py:698]: half_even is tested by existing test cases

Comment (isVoid) [reply python/cudf/cudf/tests/test_series.py:698]: Oops, just saw the PR is just about `half_up`. This is good.

**Review by isVoid — APPROVED** [review]

**Review by marlenezw — APPROVED** [review]

File: python/cudf/cudf/_lib/round.pyx [hunk 2]
```diff
@@ -29,13 +29,16 @@ def round(Column input_col, int decimal_places=0):
 
     cdef column_view input_col_view = input_col.view()
     cdef unique_ptr[column] c_result
-
+    cdef cpp_rounding_method c_how = (
```
[the remark this answers is not in the document]
Comment (shwina) [reply python/cudf/cudf/_lib/round.pyx:34]: Fixed

**Comment by shwina** [comment]
@gpucibot merge

**Pull request merged**